main.scss

```diff
@@ -35,11 +36,7 @@ body.active-info{
         visibility: visible; opacity: 1;
     }
     #instructions{ 
-        visibility: visible; opacity: 1;
-        width: 70%;
-        min-width: 25rem;
-        max-width: 50rem;
-        height: max-content;    
+        visibility: visible; opacity: 1;    

```

Comment:
Try to create a block whose size also decreases when crossed just for animations instead of just vanishing